R/scale-manual.r

```diff
@@ -56,6 +56,12 @@ scale_size_manual <- function(..., values) {
 #' @rdname scale_manual
 #' @export
 scale_shape_manual <- function(..., values) {
+  if (is.character(values) && nchar(values[1]) > 1) {
```

Comment:
I think it would be better to incorporate the existing behaviour for single letters in to `translate_shape_string()`. An easy way do that would be to combine `pch_table` with `setNames(letters, letter)`